apps/test/react/package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "dev:not-run": "vite",
     "build-only": "vite build",
-    "lints": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "yarn eslint --max-warnings 0 && yarn prettier --check",
+    "lint:fix": "yarn eslint --fix && yarn prettier --write",
+    "eslint": "eslint --ext .tsx,.ts --report-unused-disable-directives",
+    "prettier": "prettier --loglevel warn --no-error-on-unmatched-pattern \"**/*.{ts,tsx}\"",
```

Comment:
we do not need any linting here, our tests have their own linting, this is only placeholder for running tests, IMHO we should remove those so there would not be confusion